wave_lang/kernel/wave/scheduling/prefetch_scheduling.py

```diff
@@ -7,6 +7,8 @@
 from enum import Enum
 from typing import Sequence
 
+from ..schedule_reordering import AttentionOperationType
```

Comment:
Thoughts on moving attentionOperationType to https://github.com/iree-org/wave/blob/main/wave_lang/kernel/wave/utils/classes.py for a cleaner dep chain?

Reply:
sure, I can move it there.